packages/core/src/components/Table/TableRow/TableRow.tsx

```diff
@@ -171,43 +91,75 @@ export const HvTableRow = forwardRef<HTMLElement, HvTableRowProps>(
 
     const TableRow = useMemo(() => StyledTableRow(Component), [Component]);
 
+    const even = getVarValue(theme.table.rowStripedBackgroundColorEven);
+
+    const odd = getVarValue(theme.table.rowStripedBackgroundColorOdd);
```

Comment:
I'm worried about the performance implications here...
We're running `getVarValue`, which manipulates the DOM to get the value, for every render of every (potentially many) row...

Should we be memoizing this somehow?